libr/core/cmd_anal.c

```diff
@@ -593,6 +593,8 @@ static void core_anal_bytes(RCore *core, const ut8 *buf, int len, int nops, int
 				if (p && *p && strcmp (p, "null"))
 					r_cons_printf ("\"stack\":\"%s\",", p);
 			}
+			if (op.stackptr != 0)
```

Comment:
Sure - though the conditionals above here (e.g. `if (hint && hint->jump != UT64_MAX)`) don't have braces either, I just mimicked that.